src/Hooks.cs

```diff
@@ -31,11 +51,48 @@ private static void Postfix(int saveSlot)
 
         foreach (var mod in mods)
         {
+            mod.LoadSaveData(saveSlot);
             mod.LoadOnceSaveData(saveSlot);
```

Comment:
Per the comments on the comment page I think we should load once-data first, there's a pretty reasonable chance that the mutable data can have some dependence on the immutable data, but the other cannot be true by definition (since once-data has to be saved as the file starts and can't change after)